rviz_common/src/rviz_common/add_display_dialog.cpp

```diff
@@ -113,7 +113,7 @@ bool isSubtopic(const std::string & base, const std::string & topic)
   }
 
   std::string query = topic;
-  while (query != "/") {
+  while (query != "" && query != "/") {
```

Comment:
I checked this and see no reason for this change - both versions seem to work equivalently as far as I can test. Andreas is on holiday till next week, so I'd suggest just removing this. If there is an issue later on, we can still fix this.

Reply:
If I recall correctly I ended up in an infinite loop without this change.
So if it ever happens that the add display dialog is not opening, this may be the reason.
But for now we can leave it out until it occurs again.